### PR TITLE
feat(page-menu): provide page with menu template - INNO-749

### PR DESCRIPTION
**Pull request by yhuard**
# PR description

Adds a new "page with menu" variant. Also tries to fix the navigation menu responsive behavior (still under discussion)

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [x] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [x] I have checked the dependencies
- [x] I have given the fractal status “ready” to my component
- [x] ~~I have declared `@define mycomponent` in the SCSS file~~
- [x] ~~I have specified `margin: 0;` on the CSS component~~
- [x] I have provided tests
- [x] I follow the naming guidelines
- [x] ~~the component supports composition~~
- [x] ~~there are no hardcoded strings (all content come from the context)~~
- [x] I have filled the README.md file (at least a few lines)
- [x] ~~My component is listed in the root README~~
- [x] My PR has the right label(s)

### CODE REVIEW

File: framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig [hunk 1]
```diff
@@ -39,33 +39,31 @@
...
-                    <li class="ecl-navigation-menu__item"><a href="{{ _child_link.href }}" class="{{ _child_link_class_name }}">{{ _child_link.label }}</a></li>
-                  {% endfor %}
-                </ul>
-              {% endif %}
-            </li>
-          {% endfor %}
-        {% endblock %}
-      </ul>
-    </div>
+    <button class="ecl-navigation-menu__toggle ecl-navigation-menu__hamburger ecl-navigation-menu__hamburger--squeeze" aria-controls="nav-menu-expandable-root" aria-expanded="false">
```
Comment (kalinchernev) [inline framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig]: bs_win7_ie_10 0

Tiny issue on placement

Comment (kalinchernev) [inline framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig]: The search is also a bit off ...

Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig]: Testing on IE = cheating :)

File: framework/_partials/_preview-splash-page.twig [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js">
```
Comment (yhuard) [inline framework/_partials/_preview-splash-page.twig]: Splash page -> you choose the language -> the language is not yet defined. That was my logic

File: framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig (hunk 1 above)
Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig]: Indeed. I'll add some more tests

File: framework/components/ecl-navigation/ecl-navigation-menus/_horizontal-menus.scss [hunk 3]
```diff
@@ -42,6 +82,27 @@
...
+    left: 0;
+    padding: map-get($ecl-spacing, 'xs') 0 map-get($ecl-spacing, 's');
+    position: absolute;
+    top: 100%;
+    width: 100%;
+  }
+}
+
+.ecl-navigation-menu__group::before {
+  background-color: #f5f5f5;
```
Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-menus/_horizontal-menus.scss]: Thanks!

File: framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig (hunk 1 above)
Comment (kalinchernev) [inline framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig]: It's not so bad :)

Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig]: For the record, while adding tests on mobile, I've stumbled upon a display issue related to https://github.com/philipwalton/flexbugs#9-some-html-elements-cant-be-flex-containers. Will be fixed in the next commit :)

Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-menus/ecl-navigation-menus.twig]: Mobile and tablet are the same in fact, no need to test both

**Comment by kalinchernev** [comment]
Regression on FF:
firefox_regression

**Comment by Fefaine** [comment]
videoooo 📹

**Pull request merged**